php/utils.php

```diff
@@ -1057,7 +1057,7 @@ function glob_brace( $pattern, $dummy_flags = null ) {
 					. substr( $pattern, $p, $next - $p )
 					. substr( $pattern, $rest + 1 );
 
-		if ( ( $result = glob_brace( $subpattern ) ) ) {
+		if ( $result = glob_brace( $subpattern ) ) {
```

Comment:
This is going to merge conflict with https://github.com/wp-cli/wp-cli/pull/4389

Reply:
Let #4389 take priority, and I'll fix conflicts after its merged.